datasets/chorus-dt/config.ini.dist

```diff
@@ -2,4 +2,4 @@
 port = 8013
 
 [api_keys]
-gmap =
+gmap = AIzaSyCDQTRw_KOHlOOWAuaQl_nZ61IfU9Phvnc
```

Comment:
Nope :)
We shouldn't have the gmap key here.
Can you delete it, I will replace the token.